paper-input-container.html

```diff
@@ -453,20 +458,21 @@
 
     _onBlur: function() {
       this._setFocused(false);
+      this._handleValue(this._inputElement, true);
     },
 
     _onInput: function(event) {
-      this._handleValue(event.target);
+      this._handleValue(event.target, true);
     },
 
     _onValueChanged: function(event) {
-      this._handleValue(event.target);
+      this._handleValue(event.target, true);
     },
 
-    _handleValue: function(inputElement) {
-      var value = inputElement[this._propertyForValue] || inputElement.value;
+    _handleValue: function(inputElement, validateNow) {
```

Comment:
The alternative is to split `handleValue` up and always call (whenever we passed `true`):
```
this._handleValue(event.target);
this._handleAutoValidate(event.target);
```

Do you think that's better? I'm ambivalent (copy pasting is bad vs booleans are bad)